### PR TITLE
update badges

### PR DESCRIPTION
**Pull request by mbernasocchi**
remove gitter
add discussions and sponsor

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 9b4c93f21d140714f9d81633806e3cf281fa3f0d:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2849-98a4df0a2befa18cd2a2f77b1f3fe4b5e6f77739-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2849-98a4df0a2befa18cd2a2f77b1f3fe4b5e6f77739-arm-android.apk), [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2849-98a4df0a2befa18cd2a2f77b1f3fe4b5e6f77739-arm-neon-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2849-98a4df0a2befa18cd2a2f77b1f3fe4b5e6f77739-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2849-98a4df0a2befa18cd2a2f77b1f3fe4b5e6f77739-x86-android.apk)

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [![Read the documentation](https://img.shields.io/badge/Read-the%20docs-green.svg)](https://docs.qfield.org/)
-[![Join the chat at https://gitter.im/opengisch/QField](https://badges.gitter.im/opengisch/QField.svg)](https://gitter.im/opengisch/QField?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![GitHub Discussions](https://img.shields.io/github/discussions/opengisch/qfield)](https://github.com/opengisch/QField/discussions)
+[![](https://img.shields.io/static/v1?label=Sponsor&message=%E2%9D%A4&color=%23fe8e86)](https://github.com/sponsors/opengisch)
```
Comment (nirvn) [inline README.md:3]:
```suggestion
[![Support QField](https://img.shields.io/static/v1?label=Sponsor&message=%E2%9D%A4&color=%23fe8e86)](https://github.com/sponsors/opengisch)
```

Comment (m-kuhn) [reply README.md:3]:
```suggestion
[![](https://img.shields.io/static/v1?label=Support%20QField&message=%E2%9D%A4&color=%23fedeff)](https://github.com/sponsors/opengisch)
```

Comment (m-kuhn) [reply README.md:3]: @nirvn what do you think?

Comment (nirvn) [reply README.md:3]: Not a bad idea sir, looks good (better than the orange-ish light red that was there before)

Comment (m-kuhn) [reply README.md:3]:
```suggestion
[![](https://img.shields.io/static/v1?label=Support%20QField&message=%E2%9D%A4&color=%23fedeff)](https://github.com/sponsors/opengisch)
[![](https://img.shields.io/static/v1?label=Contribute%20QField&message=💪&color=%23fedeff)](#contribute)
```

**Review by nirvn — APPROVED** [review]

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 7cc246e368fa632454d4f8646ecc6b63f4cba035:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2849-cdaa670b76685972119d0a354d3c7e9d448c83b2-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2849-cdaa670b76685972119d0a354d3c7e9d448c83b2-arm-android.apk), [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2849-cdaa670b76685972119d0a354d3c7e9d448c83b2-arm-neon-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2849-cdaa670b76685972119d0a354d3c7e9d448c83b2-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2849-cdaa670b76685972119d0a354d3c7e9d448c83b2-x86-android.apk)

**Pull request merged**